### DISCUSSION TITLE
面板的登录状态缓存机制好像有点问题

### DISCUSSION THREAD

**Discussion by godcop** [1Panel-dev/1Panel#5423] [Q&A]
### 1Panel 版本

v1.10.10-lts

### 问题描述

两台服务器分别安装面板，但使用同一个顶级域名进行反向代理绑定，登录其中一个面板，会让另一个面板的登录状态失效，也就是说两个面板无法同时登录状态

### 重现步骤

如，两个面板分别绑定域名one.test.com、two.test.com，这两个面板无法同时处于登录状态

### 期待的正确结果

两个面板应互不影响，同时处于登录状态

**Comment by wanghe-fit2cloud** [comment]
我们在本地环境部署了 1Panel 最新版本，通过 Chrome 浏览器访问后未发现问题。请尝试清除浏览器缓存或更换其他浏览器，再试一下是否仍然存在该问题。

**Comment by QYG2297248353** [comment]
岂止，有的时候不知道哪根经抽风了密码明明正确，总是登录失败，换个浏览器就好了，缓存机制肯定有问题，但是又不是频发
